site.yml

```diff
@@ -266,6 +266,7 @@
   roles:
     - role: 'openstack-network'
       tags: ['openstack', 'setup']
+      when: neutron.networks|length > 1 
```

Comment:
Conceivably there could be exactly one network which is external, can we test for the same condition we use in openstack-network: a network named external or make that role complete without error when there are not networks named external.